main/settings.py

```diff
@@ -85,6 +85,16 @@ def get_var(name, default):
 
 SECURE_SSL_REDIRECT = get_var('BOOTCAMP_SECURE_SSL_REDIRECT', True)
 
+ZENDESK_CONFIG = {
+    "HELP_WIDGET_ENABLED": get_bool(
+        "ZENDESK_HELP_WIDGET_ENABLED",
+        False,
+    ),
+    "HELP_WIDGET_KEY": get_string(
+        "ZENDESK_HELP_WIDGET_KEY",
+        "d99f12ec-89dd-4111-b5ea-7b36d01bba24",
```

Comment:
I forgot, to which that default value of widget_key is pointing to ?

Reply:
Javascript one that peter mentioned on the channel.

Reply:
<!-- Start of mitbootcamps Zendesk Widget script -->
<script id="ze-snippet" src="https://static.zdassets.com/ekr/snippet.js?key=d99f12ec-89dd-4111-b5ea-7b36d01bba24"> </script>
<!-- End of mitbootcamps Zendesk Widget script -->